examples/window/window_settings.rs

```diff
@@ -8,6 +8,12 @@ fn main() {
             width: 500.,
             height: 300.,
             vsync: true,
+            resizable: true,
```

Comment:
Not sure if this should be merged into WindowResizeConstraints - however that would be in a seperate PR anyway. 

I suspect it shouldn't be just because we don't want to merge too many settings into one command.